edge-modules/azure-monitor/src/MetricsScrapeAndUpload.cs

```diff
@@ -15,12 +15,14 @@ class MetricsScrapeAndUpload : IDisposable
     {
         private readonly MetricsScraper scraper;
         private readonly IMetricsPublisher publisher;
+        private readonly Func<Tuple<Metric, string>, Tuple<Metric, string>> adjustMetric;
```

Comment:
I don't think you need to specify Tuple everytime , simply this should work?

```private readonly Func<(Metric, string), (Metric, string)> adjustMetric```

Reply:
Wasn't aware of this syntax.  Thx.